hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/Create2OperationSuite.java

```diff
@@ -549,13 +549,6 @@ contract, GET_BYTECODE, asHeadlongAddress(factoryEvmAddress.get()), salt)
                                 .nodePayment(ONE_HBAR)),
                         sourcing(() -> setExpectedCreate2Address(
                                 contract, salt, expectedCreate2Address, testContractInitcode)),
-                        sourcing(() -> contractCall(contract, DEPLOY, testContractInitcode.get(), salt)
```

Comment:
I'm always - _always!_ - happy to get rid of code but why isn't this necessary anymore?   (And below at L673-679)